.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
```suggestion
  - package-ecosystem: "gomod"
```

Reply:
I think we may want both, I'll push a commit to add ``gomod`` instead of swapping it.